utils/docker/Dockerfile.centos.7

```diff
@@ -180,28 +176,30 @@ RUN if [ "x$BULLSEYE" != "x" ]; then \
       rm -rf bullseye; \
     fi
 
-
 # force an upgrade to get any newly built RPMs
 ARG CACHEBUST=1
 RUN yum -y upgrade --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury
 
-# Switch to new user
-#USER $USER
-WORKDIR /home/$USER
-
-# set NOBUILD to disable git clone & build
-ARG NOBUILD
-
-# Fetch DAOS code
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-    git clone --recurse-submodules https://github.com/daos-stack/daos.git; fi
-WORKDIR /home/$USER/daos
+# copy DAOS tree
+WORKDIR /home/daos
+COPY ./ .
 
 # Build DAOS & dependencies
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-    scons --build-deps=yes install PREFIX=/opt/daos; fi
+RUN scons --build-deps=yes -j 4 install PREFIX=/opt/daos
 
 # Set environment variables
 ENV LD_LIBRARY_PATH=/opt/daos/lib:/opt/daos/lib64:$LD_LIBRARY_PATH
 ENV PATH=/opt/daos/bin:$PATH
 ENV FI_SOCKETS_MAX_CONN_RETRY=1
+
+# Copy example config files to /opt/daos
+COPY utils/config/examples /opt/daos
+
+# Build java HDFS bindings
+# Currently disabled since it is failing
+RUN pushd src/client/java && \
+#    mvn clean install -T 1C -DskipITs -Ddaos.install.path=/opt/daos && \
```

Comment:
This was working in my PR: https://github.com/daos-stack/daos/runs/1582021259?check_suite_focus=true

It was only the Leap 15 build that was failing which I opened [DAOS-6374](https://jira.hpdd.intel.com/browse/DAOS-6374) for.

Since you are doing this in the Dockerfile now, and not as part of the GitHub Action matrix, it seems we should be able to enable it where it works and disable it where it doesn't?

Reply:
It has been broken by recent changes/landings. I am working with Jiafu to fix it.